src/routes/organization.rs

```diff
@@ -52,13 +51,8 @@ pub async fn delete(
         )));
     }
 
-    organization::delete(db, id)
-        .await
-        .map_err(|e| Error::ServerError(Json(e.to_string().into())))?;
-
-    remove_dir_all(Path::new("content/").join(id))
-        .await
-        .map_err(|e| Error::ServerError(Json(e.to_string().into())))?;
+    organization::delete(db, id).await?;
+    remove_dir_all(Path::new("content/").join(id)).await?;
```

Comment:
# Are you kinding me?

I feel very scared, luckily these codes is not in stable version.

Reply:
Leave it here...I would refact this later to fit your crazy cleaning.